### DISCUSSION TITLE
Sydney and Greater NSW Community

### DISCUSSION THREAD

**Discussion by TheITJuggler** [meshtastic/Discussions#10] [South Asia/APAC]
Hey ho.. let's go!

**Comment by rthorntn** [comment]


**Reply by Soupbones** [reply]
Sure is, growing weekly, but very slowly.

**Reply by yamez1969** [reply]
I'm picking up nodes when I travel via train to the Sydney CBD. I think that some 20km plus, no hop contacts are made by airplane scatter.

**Reply by Soupbones** [reply]
Where you travelling from?

**Reply by TheITJuggler** [reply]
I pick up 2 nodes just near North Sydney when on the bus.

**Reply by yamez1969** [reply]
Traveling via train from Thirroul to Bondi Junction (BJ). My node at BJ sits on a window sill, facing a walled courtyard with the open sky visible, so is RF compromised. There is a node nearby within 300M that has connected. However I have got 20km, no hop contacts from various locations that I attribute to aero plane scatter. Kingsford Smith Airport is nearby, with low altitude take offs.

**Comment by Soupbones** [comment]
> Hey ho.. let's go!

Ok, replying from the discource they booted out new conversation from (still don't know why) but I think your right in regards to Mt Kaputar Meshtastic distance record.  I looked at 'HeyWhatsThat' again and their Earth curvature looks a bit wrong. So I checked it using Google earth and it will just be cut off from Earth Curviture. So we need to find a bigger mountain to TX from. Somehow I don't think we can get the record or beat 315Km here in Australia. We don't have enough mountains.

**Comment by Soupbones** [comment]
Well this chat page idea died out pretty quickly.

**Comment by TheITJuggler** [comment]
My apologies - I've been otherwise engaged and haven't been updating here.

**Reply by Soupbones** [reply]
All good. Is the BBS coming back any time?

**Comment by TheITJuggler** [comment]
I'm aiming to have it back online in the coming days. One of the features
I'm looking for is weather alerts

**Comment by WJ4IoT** [comment]
Hi everybody, living in the Netherlands but for two months my home will be on the bus in Australia, so travel around and mostly roundish the east coast and some few 100 miles land inwards. Though changes are limited I might have contact with some of you (short: WJPM). Hope it happens.

**Comment by WJ4IoT** [comment]
Hi everybody, living in the Netherlands but for two months my home will be on the bus in Australia, so travel around and mostly roundish the east coast and some few 100 miles land inwards. Though changes are limited I might have contact with some of you (short: WJPM). Hope it happens.

**Reply by Soupbones** [reply]
Hello! You will have a great time in Australia. So much tomsee and do. As for Meshtastic, Melbourne, Sydney and Brisbane all have active Meshes. The Sydney mesh is a bit of a mess att the moment as one of our nodes up in the mountains is down, but will be back up. Sydney's Mesh is still growing. Brisbane has a big mesh, and Canberra has one of the best Mesh's in Australia, very well though out with awesome coverage all over. The Sydney Mesh has its own MQTT so make sure you have MQTT on in Sydney. Jump on, say hello and if I see you I'll say hello 👋 too!

**Reply by WJ4IoT** [reply]
Planning only to use RF on my node, MQTT is a bit of cheating. There are better text services. I want to snif MQTT at home straight from the server and only protobufs, Can do this in NodeRed but encounter a small problem and that is that the decode function of @meshtastic/node-red-contrib-meshtastic like to collect but not drop data so it is flooding my drive in no-time to zero bytes available. Must be doing something wrong.

**Comment by tomlister** [comment]
I can't pick up any nodes in Sydney sadly.

**Reply by Soupbones** [reply]
Not sure why not, there are tonnes of nodes across the Sydney Metro area and Wollongong region.

**Reply by tomlister** [reply]
could be my shitty setup

**Reply by tomlister** [reply]
nevermind. started getting some.

**Comment by tomlister** [comment]
RAKMBS0A (8a087cd9) is the GOAT rn, found so many other nodes through it

**Comment by WJ4IoT** [comment]
My camper trip is starting upcomming week and from Melbourne it will go round and up in 6 weeks towards Brisbane. So covering a (very) wide area, My T114 will be always on. At home I will be sniffing MQTT with Node-Red on "msh/ANZ/#". 

Where and when I will be I do not know but I hope to pick up some nodes along the way and say hello to you.
Attached a rough map of POI's. Suggestion are welcome.

![AUSTRALIE_SMALL](https://github.com/user-attachments/assets/d18d69d5-ffc2-45c5-84c8-8da134cf75a5)

BTW Tackled the problem of the original palette which filled disc space like a bug on steroid, a working decrypt came from a French guy.

**Reply by Soupbones** [reply]
Well, it's been a few weeks. I haven't seen your node in Sydney at all. What is your node name? Also, you won't find many of us on the default Meshtastic MQTT as we have our own private MQTT set up for Sydney's great metro. Here are the details.
MQTT: Enabled
MQTT Client Proxy: Enabled
Root topic: msh/ANZ/NSW 
Just so you know, using only msh/ANZ is blocked on this server.
Address: [nsw.mqtt.meshtastic.au](http://nsw.mqtt.meshtastic.au/?fbclid=IwZXh0bgNhZW0CMTAAAR1K5HcFaizRqefBOLKmV2JSFthadZqUESj_CKN0PWdCwB0ZVlVfw9550xE_aem_aX7bTDcLV9rBoWXXlvXmRw)
username: nswmesh
password: nswmesh
TLS Enabled: no
In Channels, in the Primary channel, ensure you have both MQTT Uplink and Downlink enabled.

**Reply by WJ4IoT** [reply]
That is because I am not yet close to Sidney. At the moment in Jervis Bay and after the weekend touring in the Blue mountains. How close I get to Sydney I do not know yet. Since the start I have seen 6 other nodes in the Melbourne area, one tried to contact me what I missed. Since then it is me, myself and I 😁. Perhaps I should activate full MQTT on my node but still think this is a bit of make-believe. I do use the uplink occasionally to put my location on the map: short node name WJPM.

Is the MQTT server mentioned syncing its data with the main server?

**Reply by Soupbones** [reply]
Travelling from Jervis bay to Blue Mountains you will most likely cut through the SW of the Sydney Metro area. Plenty of Mesh nodes out that way, and the Blue Mountains is directly West of Syd. Nodes in the mountains help the rest of Syd link up. VK2KTJ also has a node right up the top of the Blue Mountains at Blackheath. He doesn't really get anyone apart from nodes on MQTT so he'd be happy too get you.

**Reply by Soupbones** [reply]
Where are you now WJ4I? A bunch of us saw you call out on the Longfast yesterday a couple of times but no real replies.

**Comment by tomlister** [comment]
turns out the wall near the ledge of my balcony is acting as a reflector lol. when my node is placed half a wavelength away from it i can pick stuff up but any further away or closer to the wall it can't. kinda epic.

**Comment by TheITJuggler** [comment]
Yea, the nsw mqtt server uplinks to both the Meshtastic server and the liamcottle server. Sent from my iPhone - please excuse the short message.On 3 Apr 2025, at 09:41, WJ ***@***.***> wrote:﻿
That is because I am not yet close to Sidney. At the moment in Jervis Bay and after the weekend touring in the Blue mountains. How close I get to Sydney I do not know yet. Since the start I have seen 6 other nodes in the Melbourne area, one tried to contact me what I missed. Since then it is me, myself and I 😁. Perhaps I should activate full MQTT on my node but still think this is a bit of make-believe. I do use the uplink occasionally to put my location on the map: short node name WJPM.
Is the MQTT server mentioned syncing its data with the main server?

**Comment by WJ4IoT** [comment]
We are now settled in Warragamba and my node is with MQTT disabled since this morning, picked up some nodes during travel like GRHI and a few others. Here there perhaps to much trees to to get a signal.Sent from Android deviceOp 5 apr 2025 09:48 schreef David ***@***.***>:
Where are you now WJ4I? A bunch of us saw you call out on the Longfast yesterday a couple of times but no real replies.

**Comment by WJ4IoT** [comment]
Arrived yesterday on Umina Beach, looks there is a node TMZT very nearby but got a public key mismatch, this has happen to me more often so I wonder people using a different PSK? I also notice its role is router, mayby unmanaged and on high ground.

**Comment by e1kjay** [comment]
Hi all, just got up on the Sydney mesh from Bankstown this week. My nodes have good LoS to the south/south-west so pleased to discover there are some good repeaters out that way. 

Connection is still pretty dependent on ionosphere which seems to allow for best range between 01:30-11:30. After that my node more or less drops off the mesh, though this window was shorter before I made a 1/4 wave ground plane for my outdoor node (to my relief this took me from 3 nodes to 80 overnight). I’m experimenting with more directional antennas to see if I can get a consistent connection to repeaters in the south-west. 

I enabled MQTT with the config  @Soupbones mentioned above but yet to see any activity beyond what I’ve got over RF. I’d love to link up with the Sydney mesh community so if there are any other places people are talking and planning please let me know! 

VK2AEU
Short:31kj

**Reply by OzCleric** [reply]
you should come join the local Facebook group; this discord thread is not very active.

https://www.facebook.com/share/g/1Drm4fGDqp/

There are 119 members in that Sydney Meshtastic and Meshcore FB group; including myself and @Soupbones

**Reply by Soupbones** [reply]
> Hi all, just got up on the Sydney mesh from Bankstown this week. My nodes have good LoS to the south/south-west so pleased to discover there are some good repeaters out that way.
> 
> Connection is still pretty dependent on ionosphere which seems to allow for best range between 01:30-11:30. After that my node more or less drops off the mesh, though this window was shorter before I made a 1/4 wave ground plane for my outdoor node (to my relief this took me from 3 nodes to 80 overnight). I’m experimenting with more directional antennas to see if I can get a consistent connection to repeaters in the south-west.
> 
> I enabled MQTT with the config @Soupbones mentioned above but yet to see any activity beyond what I’ve got over RF. I’d love to link up with the Sydney mesh community so if there are any other places people are talking and planning please let me know!
> 
> VK2AEU Short:31kj

Hi VK2AEU. I am getting your node out here in the far SW, great at the moment. You may need to tidy up your node name as I can't see your full callsign. Maybe I suggest changing the short name to 2AEU.  Jump on the Facebook page if you want and introduce yourself and your set-up. The FB page seems to have spurts of activity, then goes quiet most of the week. So I encourage people, when I can, to at least put up signal reports if they are testing their node capabilities. This, I think, helps others understand the range of their nodes are getting out.
 I've noticed with the high humidity we have at the moment the mesh is not connecting as well. Hopefully that will change soon. I just cleaned out the node data base of one of my home nodes and it had 179 nodes in it. So the Mesh is certainly getting huge!

**Comment by coaxk** [comment]
could anyone please send the active Discord link/invite?
